src/main/webapp/app/shared/modal/modify-cancer-type-modal.store.ts

```diff
@@ -38,13 +43,51 @@ export class ModifyCancerTypeModalStore {
     this.isErrorIncludedAndExcluded = false;
   }
 
-  setIncludedCancerTypes(cancerTypes: CancerTypeSelectOption[]) {
+  private setIsErrorDuplicate(
+    included: CancerTypeSelectOption[],
+    excluded: CancerTypeSelectOption[],
+    allCancerTypes: Tumor[],
+    cancerTypeToEditUuid?: string,
+  ) {
+    const includedCancerTypes = included.map(option => getCancerTypeFromCancerTypeSelectOption(option));
+    const newCancerTypeIncludedParts = getCancerTypesName(includedCancerTypes, true, ',').split(',');
+    newCancerTypeIncludedParts.sort();
+    const newCancerTypeIncluded = newCancerTypeIncludedParts.join(',');
```

Comment:
Looks like the logic to get the "sorted" cancer type name is duplicated in 4 spots. Maybe we can move it to utils?

Reply:
I decided to just make it a private method since I don't see it being to useful elsewhere